sdk/keyvault/keyvault-certificates/src/lro/create/operation.ts

```diff
@@ -139,7 +139,7 @@ function toString(this: CreateCertificatePollOperation): string {
  * @param [state] A poll operation's state, in case the new one is intended to follow up where the previous one was left.
  */
 export function makeCreateCertificatePollOperation(
-  state: CreateCertificatePollOperationState
+  state: CreateCertificatePollState
```

Comment:
PollerState?

Reply:
I guess I might be confused about PollThing and PollerThing

Reply:
We called the state of the poller: PollOperationState. I reduced it to PollState. Should I change it?

Reply:
Previously you said "let me see if I can rename them all to be just *PollerState instead of *PollOperationState". Is that different than here?

Reply:
Oh, I was thinking on PollState when I wrote that. Which one do you prefer? I prefer PollState since it's PollOperationState shortened.

Reply:
PollState to me sounds like something that says "the state our poll is currently in" but I think you're trying to say "the state the current operation is in" rather than saying something about the poll.

Do Pollers have any other public state associated with them?

Reply:
no, all the poller state is encapsulated in the poller's operation state

Reply:
I'll resolve this for now.